runtime/mangata-kusama/src/lib.rs

```diff
@@ -124,6 +126,7 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
+	migration::asset_register::MigrateToXykMetadata,
```

Comment:
the migration has to be part of the runtime, that's where the types are defined for asset-registry
otherwise we would need to introduce our specific runtime types into the orml library